gensim/scripts/word2vec2tensor.py

```diff
@@ -57,7 +57,7 @@ def word2vec2tensor(word2vec_model_path, tensor_filename, binary=False):
     outfiletsvmeta = tensor_filename + '_metadata.tsv'
 
     with open(outfiletsv, 'w+') as file_vector:
-        with open(outfiletsvmeta, 'w+') as file_metadata:
+        with open(outfiletsvmeta, 'wb+') as file_metadata:
```

Comment:
Why did this need? python3.5 problem?

Reply:
`%run ../../gensim/scripts/word2vec2tensor.py -i doc_tensor.w2v -o movie_plot` This line of code in `Tensorboard_visualizations.ipynb` fails without this change

Reply:
Can you check this script with other python versions (`2.7` or `3.6`)? This is really suspicious.

Reply:
Okay